code/game/objects/items/tools/multitool.dm

```diff
@@ -53,6 +53,7 @@
 	var/rangealert = 8	//Glows red when inside
 	var/rangewarning = 20 //Glows yellow when inside
 	origin_tech = "magnets=1;engineering=2;syndicate=1"
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Not sure if there are other ways to check whether a multitool is an AI detector or not, but this will enable anyone to check for it by simply putting it into a box! Just a heads up in case you want to avoid it

Reply:
Considering you can already just ask the AI to fly its camera over you as Security I dont think its that massive of a deal.